py/dml/ctree.py

```diff
@@ -3534,6 +3536,19 @@ def ctype(self):
     def read(self):
         return "(%s).id" % (self.traitref.read(),)
 
+class LogObjectFromObjIdentity(Expression):
+    priority = dml.expr.Apply.priority
+    @auto_init
+    def __init__(self, site, identity):
+        crep.require_dev(site)
+
+    def ctype(self):
+        return TNamed('_conf_object_t *')
```

Comment:
what's a `_conf_object_t`?

Reply:
_very_ good question.

I'm to used to dmllib's naming scheme.